src/fetcher/create-taxonomy.ts

```diff
@@ -93,6 +93,7 @@ function collectType(
 }
 
 function getAlias(child: { alias?: string | null; id: number }) {
+  // TODO: do we still need this?
   if (!child.alias || hasSpecialUrlChars(child.alias)) return `/${child.id}`
   else return child.alias
```

Comment:
@Entkenntnis do you remember what the `hasSpecialUrlChars` check was for?

Reply:
As far as I remember next.js-csr had some struggles with special chars.